src/commander.c

```diff
@@ -599,6 +599,11 @@ static void commander_process_seed(yajl_val json_node)
     } else if (ret == DBB_ERROR_MEM) {
         commander_fill_report(cmd_str(CMD_seed), NULL, DBB_ERR_MEM_ATAES);
     } else if (ret == DBB_OK) {
+        // unset U2F bit = enable U2F by default
+        uint32_t flags = memory_extflag_read();
+        flags &= ~(EXT_FLAG_U2F);
+        memory_extflag_write(flags);
+
```

Comment:
IMO, better to keep U2F enable/disable independent of reseeding (i.e. drop this part - I am refactoring now, so can drop it in a new commit). Reasoning: There is a use case of switching between a 'savings' and 'checking' wallet - someone who disables U2F probably doesn't want to disable U2F every time the wallet is switched.